x-pack/legacy/plugins/code/public/monaco/monaco_diff_editor.ts

```diff
@@ -26,6 +26,21 @@ export class MonacoDiffEditor {
         enableSplitViewResizing: false,
         renderSideBySide: this.renderSideBySide,
         scrollBeyondLastLine: false,
+        readOnly: true,
+        minimap: {
```

Comment:
I also had no luck hiding the overview. However, it might be worth adding `overviewRulerBorder: false` to cut down slightly on the noise that it adds.